pkg/cmd/server/server.go

```diff
@@ -397,47 +406,49 @@ func durationMin(a, b time.Duration) time.Duration {
 }
 
 func (s *server) runControllers(config *api.Config) error {
-	glog.Infof("Starting controllers")
+	s.logger.Info("Starting controllers")
 
 	ctx := s.ctx
 	var wg sync.WaitGroup
 
 	cloudBackupCacheResyncPeriod := durationMin(config.GCSyncPeriod.Duration, config.BackupSyncPeriod.Duration)
-	glog.Infof("Caching cloud backups every %s", cloudBackupCacheResyncPeriod)
+	s.logger.WithField("cloudBackupCacheResyncPeriod", cloudBackupCacheResyncPeriod).Info("Setting cloud backup cache sync period")
```

Comment:
I'd change cloudBackupCacheResyncPeriod to one of period, resyncPeriod, duration. Trying to avoid repetition.